Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM alpine:3.12
 
 RUN apk --no-cache add \
+    bash \
```

Comment:
This is not really needed for this PR. I added it while I tested to install helm unittest in the container.
Looks like some plugins need bash to be able to use `helm plugin install`.

Adding it here makes it easier to extend the container.